core/tests/protractor_desktop/contributorDashboard.js

```diff
@@ -140,6 +154,91 @@ describe('Contributor dashboard page', function() {
       await users.logout();
     });
 
+  it('should allow the users to use the copy tool', async function() {
```

Comment:
Have you got 5 straight successful runs of the suite that contains this test? If so, please attach the links of those in the PR description?

Reply:
Done

Reply:
Done